cpu/esp32/esp-eth/esp_eth_netdev.h

```diff
@@ -37,11 +37,11 @@ typedef struct
 {
     netdev_t netdev;                    /**< netdev parent struct */
 
-    uint8_t rx_len;                     /**< number of bytes received */
-    uint8_t rx_buf[ETHERNET_DATA_LEN];  /**< receive buffer */
+    uint16_t rx_len;                     /**< number of bytes received */
+    uint8_t  rx_buf[ETHERNET_DATA_LEN];  /**< receive buffer */
 
-    uint8_t tx_len;                     /**< number of bytes in transmit buffer */
-    uint8_t tx_buf[ETHERNET_DATA_LEN];  /**< transmit buffer */
+    uint16_t tx_len;                     /**< number of bytes in transmit buffer */
```

Comment:
Maybe group the length's together? this might save some bytes of RAM, since the `uint16_t`s don't cause padding then, so that the arrays are aligned.